appenders/bash/logger.sh

```diff
@@ -4,49 +4,60 @@
 # Export the variables: PULSE_COLLECTOR_HOST and PULSE_COLLECTOR_PORT for the host and port caontaing the Pulse
 # Log Collector
 # ```
-# export PULSE_COLLECTOR_HOST=host1.com
-# export PULSE_COLLECTOR_PORT=9004
+# export PULSE_LOG_COLLECTOR_HOST=edge2.valhalla.phdata.io
+# export PULSE_LOG_COLLECTOR_PORT=9004
+# export APPLICATION_NAME=pulse_test_default
 # ```
-# Then source this scrip in your own bash scrip
-# ``` 
-# source logger.sh
-# logger --category=category --level=level --message=message --threadname=threadname --application=application-name
-# ```
-set -euo pipefail
+
+declare ERROR_MESSAGE="Error! Message info is only possible for its argument."
 
 logger(){
-  if [ "$#" -lt 1 ]
+  if [ -z "$APPLICATION_NAME" ] || [ -z "$PULSE_LOG_COLLECTOR_HOST" ] || [ -z "$PULSE_LOG_COLLECTOR_PORT" ]
   then
-  echo "Usage Info: logger --category=category --level=level --message=message --threadname=threadname --application=application-name"
-  exit 1
+    echo "Error! Env variable should have PULSE_LOG_COLLECTOR_HOST, PULSE_LOG_COLLETOR_PORT, and APPLICATION_NAME"
+  else
+    timestamp=$( date -u +"%Y-%m-%dT%H:%M:%SZ")
+    echo $timestamp
+    curl -s -X POST -H 'Content-Type: application/json' -d '{"category": "","timestamp": "'$timestamp'", "level": "'$1'", "message": "'$2'", "threadName": ""}' http://$PULSE_LOG_COLLECTOR_HOST:$PULSE_LOG_COLLECTOR_PORT/v2/event/$APPLICATION_NAME
   fi
+}
 
-  while [ $# -gt 0 ];do
-  case "$1" in
-    --category=*)
-        category="${1#*=}"
-    ;;
-    --level=*)
-        level="${1#*=}"
-    ;;
-    --message=*)
-        message="${1#*=}"
-    ;;
-    --threadName=*)
-        threadName="${1#*=}"
-    ;;
-    --application=*)
-        application="${1#*=}"
-        ;;
-    *)
-        echo "Usage Info: logger --category=category --level=level --message=message --threadname=threadname"
-        exit 1
-  esac
-    shift
-  done
-
-  timestamp=$( date -u +"%Y-%m-%dT%H:%M:%SZ")
+pulse_debug(){
+  if [ "$#" -lt 1 ]
+  then
+    echo $ERROR_MESSAGE
+  else
+    echo "Pulse is debuging \"$1\" message"
```

Comment:
These messages can be enabled by a `PULSE_DEBUG` variable but they shouldn't print all the time, users won't want to see them